TheFourFifths/consus
# Admin override tff 79

### Summary

Server side code for admin override


### Checklist


Finally, is it ready to be reviewed and merged: yes :white_check_mark:

### How to Review

1. Pull
2. Check tests and code for any poor coding practices.
3. Run Server
4. Run Client
5. Get Boaty McBoatface (ID: 111111)
6. Try to check out item.
7. Modal should ask for ID (Enter: 112994 or 2013)
8. Complete Checkout

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-79](https://msoese.atlassian.net/browse/TFF-79)
- [Client Code(https://github.com/TheFourFifths/consus-client/pull/27)
